cmd/reposcaffolding/github/main.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"log"
+
+	"github.com/merico-dev/stream/internal/pkg/plugin/reposcaffolding/github"
+)
+
+// NAME is the name of this DevStream plugin.
+const NAME = "repo-scaffolding-github"
```

Comment:
As per our discussion, the name should be "github-repo-scaffolding".

After second thought, we should also include the name of the programming language as part of the plugin, because, for example, golang plugin needs to handle helm chart templates rendering, while python plugin doesn't.

Reply:
github-repo-scaffolding is right.
but for the language suffix, maybe we need to discuss it more.